documentation/projects/proposals/document_all_media_properties/20230426-implementation_plan_documenting_media_properties.md

```diff
@@ -0,0 +1,137 @@
+# Implementation Plan: Catalog
+
+<!-- See the implementation plan guide for more information: https://github.com/WordPress/openverse/tree/19791f51c063d0979112f4b9f4eeace04c8cf5ff/docs/projects#implementation-plans-status-in-rfc -->
+<!-- This template is exhaustive and may include sections which aren't relevant to your project. Feel free to remove any sections which would not be useful to have. -->
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @AetherUnbound
+- [ ] @stacimc
+
+## Project links
+
+<!-- Enumerate any references to other documents/pages, including milestones and other plans -->
+
+- [Project Thread](https://github.com/WordPress/openverse/issues/412)
+- [Project Proposal](https://docs.openverse.org/projects/proposals/document_all_media_properties/20230307-project_proposal.html#description)
+
+## Overview
+
+<!-- A brief one or two sentence overview of the implementation being described. -->
+
+A proposal to create a documentation page with description of the media
+properties collected by the Catalog.
+
+## Background
+
+The catalog collects and transforms data from many different sources. The kind
+of data collected, transformations, and the expected data shape have changed
+over time. This plan will outline the creation of the documentation page about
+the media properties in the catalog. This will help answer some of the questions
+like:
+
+### User stories
+
+- As a maintainer/contributor, I need to know the list of available properties
+  for each media type in the catalog database. ... so that?
+- As a contributor, I need to know what is the expected data type for the
+  specific property. Is it required? ... so that?
```

Comment:
I might be unfamiliar with user stories in the traditional sense, is the `... so that?` intended here or was there other information/examples that should be provided?

Reply:
Got it, thanks!